packages/broker/test/integration/plugins/operator/VoteOnSusppectNodeService.test.ts

```diff
@@ -0,0 +1,131 @@
+import { Chain } from '@streamr/config'
+
+const theGraphUrl = `http://${process.env.STREAMR_DOCKER_DEV_HOST ?? '10.200.10.1'}:8000/subgraphs/name/streamr-dev/network-subgraphs`
+
+jest.setTimeout(600000)
+
+const ADMIN_PRIV_KEY = "0x2cd9855d17e01ce041953829398af7e48b24ece04ff9d0e183414de54dc52285" // sidechain
+// const ADMIN_PRIV_KEY = "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80" // fastChain
+
+const logger = new Logger(module)
+describe('MaintainTopologyService', () => {
+    let provider: Provider
+    let adminWallet: Wallet
+    let token: TestToken
+    let streamId1: string
+    let streamId2: string
+    let streamrEnvDeployer: StreamrEnvDeployer
+    let config: Chain
+
+    // const chainURL = config.rpcEndpoints[0].url
+    // const chainURL = "http://127.0.0.1:8545"
+    const chainURL = 'http://10.200.10.1:8546'
+
+    beforeAll(async () => {
+        streamrEnvDeployer = new StreamrEnvDeployer(ADMIN_PRIV_KEY, chainURL)
+        await streamrEnvDeployer.deployEvironment()
+        const { contracts } = streamrEnvDeployer
+        config = { contracts: streamrEnvDeployer.addresses } as unknown as Chain
+
+        provider = new JsonRpcProvider(chainURL)
+        logger.debug('Connected to: ', await provider.getNetwork())
+
+        adminWallet = new Wallet(ADMIN_PRIV_KEY, provider)
+    
+        token = contracts.DATA as unknown as TestToken
+        const timeString = (new Date()).getTime().toString()
+        const streamPath1 = '/operatorclienttest-1-' + timeString
+        const streamPath2 = '/operatorclienttest-2-' + timeString
+        streamId1 = adminWallet.address.toLowerCase() + streamPath1
+        streamId2 = adminWallet.address.toLowerCase() + streamPath2
+        logger.debug(`creating stream with streamId1 ${streamId1}`)
+        await (await contracts.streamRegistry.createStream(streamPath1, '{"partitions":"1"}')).wait()
+        logger.debug(`creating stream with streamId2 ${streamId2}`)
+        await (await contracts.streamRegistry.createStream(streamPath2, '{"partitions":"1"}')).wait()
+
+    })
+    
+    it('allows to flag an operator as malicious', async () => {
+        const flagger = await createWalletAndDeployOperator(provider, config, theGraphUrl, ADMIN_PRIV_KEY)
+        logger.debug('deployed flagger contract ' + flagger.operatorConfig.operatorContractAddress)
+        const target = await createWalletAndDeployOperator(provider, config, theGraphUrl, ADMIN_PRIV_KEY)
+        logger.debug('deployed target contract ' + target.operatorConfig.operatorContractAddress)
+        const voter = await createWalletAndDeployOperator(provider, config, theGraphUrl, ADMIN_PRIV_KEY)
+        logger.debug('deployed voter contract ' + voter.operatorConfig.operatorContractAddress)
+
+        await new Promise((resolve) => setTimeout(resolve, 5000)) // wait for events to be processed
+        const flaggerClient: MockProxy<StreamrClient> = mock<StreamrClient>()
```

Comment:
No need to mock these `StreamrClient` instances imo. We could just pass real ones? `new StreamrClient()`.